owl-files/comploinc-default/README.md

```diff
@@ -1 +1,25 @@
-Place the default build output files in this directory and then open the comploinc.owl file in Protege to get what is considered to be the default content of CompLOINC (still WIP).
+Place the default modular output files in this directory and then open a `comploinc.owl`  in Protégé to get what is 
+considered to be the default content of CompLOINC (still WIP).
+
+## Variations
+For a given set of modular output files, there are several standard ways to combine the modules into singular merged 
+artefacts.
+
+For each variation, there is a set of 2 files: 
+- `catalog-v001-SOURCES-PART_MODEL.xml`
+- `comploinc-SOURCES-PART_MODEL.owl`
```

Comment:
There are now 6 variations to these files:

(LOINC | LOINC-SNOMED | both) x (primary | supplementary)